src/stores/workspace/SearchWorkspaceStore.ts

```diff
@@ -98,39 +94,28 @@ export default class SearchWorkspaceStore {
 		let initialWorkspaceState: WorkspaceInitialState = {};
 
 		if (isEventMessage(resultItem)) {
-			const requestInfo = this.searchStore.currentSearch?.request;
-			initialWorkspaceState = {
-				messages: {
-					sse: (requestInfo?.filters as MessageFilterState) || null,
-					streams: requestInfo?.state.stream || [],
-					timestampFrom: null,
-					timestampTo: timestampToNumber(resultItem.timestamp),
-				},
-				interval: SEARCH_STORE_INTERVAL,
-				layout: [0, 100],
-				timeRange: getRangeFromTimestamp(timestampToNumber(resultItem.timestamp), 15),
-			};
+			initialWorkspaceState = this.workspacesStore.getInitialWorkspaceByMessage(
+				timestampToNumber(resultItem.timestamp),
```

Comment:
Lets pass a message here also and give initialState a target so that message is highlighted and scrolled on loading